frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -96,17 +100,50 @@ export const ClusterOperatorPage: React.SFC<ClusterOperatorPageProps> = props =>
     rowFilters={filters}
   />;
 
+const OperandVersions: React.SFC<OperandVersionsProps> = ({versions}) => {
+  return _.isEmpty(versions)
+    ? <EmptyBox label="Versions" />
+    : <div className="co-table-container">
+      <table className="table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          {_.map(versions, ({name, version}, i) => (
+            <tr key={i}>
+              <td>{name}</td>
+              <td>{version}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    </div>;
+};
+
+
 const ClusterOperatorDetails: React.SFC<ClusterOperatorDetailsProps> = ({obj}) => {
   const { status, message } = getStatusAndMessage(obj);
-  return <div className="co-m-pane__body">
-    <SectionHeading text="Cluster Operator Overview" />
-    <ResourceSummary resource={obj} showPodSelector={false} showNodeSelector={false}>
-      <dt>Status</dt>
-      <dd><OperatorStatusIconAndLabel status={status} /></dd>
-      <dt>Message</dt>
-      <dd>{message || '-'}</dd>
-    </ResourceSummary>
-  </div>;
+  const versions: OperandVersion[] = _.get(obj, 'status.versions', []);
+  return (
+    <React.Fragment>
+      <div className="co-m-pane__body">
+        <SectionHeading text="Cluster Operator Overview" />
+        <ResourceSummary resource={obj} showPodSelector={false} showNodeSelector={false}>
+          <dt>Status</dt>
+          <dd><OperatorStatusIconAndLabel status={status} /></dd>
+          <dt>Message</dt>
+          <dd className="co-pre-line">{message || '-'}</dd>
+        </ResourceSummary>
+      </div>
+      <div className="co-m-pane__body">
```

Comment:
I think you want to add a `co-m-pane__body--alt` here to turn off the bottom border?